[NETBEANS-54] Module Review #194

batouxclass opened the pull request:
Fix #194 

Replaced the old license with the Apache license for spring.beans module.

batouxclass committed:
[NETBEANS-54] Module Review #194

Replaced the old license with the Apache license for spring.beans
module.

junichi11 commented:
Commit message must be `[NETBEANS-54] Module Review spring.beans`

springXMLConfig.template is a template file. Probably, the license header is the following?
```
<#--
    Licensed to the Apache Software Foundation (ASF) under one
    or more contributor license agreements.  See the NOTICE file
    distributed with this work for additional information
    regarding copyright ownership.  The ASF licenses this file
    to you under the Apache License, Version 2.0 (the
    "License"); you may not use this file except in compliance
    with the License.  You may obtain a copy of the License at
      http://www.apache.org/licenses/LICENSE-2.0
    Unless required by applicable law or agreed to in writing,
    software distributed under the License is distributed on an
    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
    KIND, either express or implied.  See the License for the
    specific language governing permissions and limitations
    under the License.
-->
```

batouxclass committed:
[NETBEANS-54] Module Review spring.beans

Applied code suggestion.

batouxclass commented:
Hi @junichi11, I just applied your code suggestion.

emilianbold commented on spring.beans/src/org/netbeans/modules/spring/beans/resources/templates/springXMLConfig.template:
```diff
@@ -1,4 +1,20 @@
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
```
newlines?

emilianbold reviewed (changes requested):
whitespace

batouxclass committed:
[NETBEANS-54] Module Review spring.beans

Added newlines.

batouxclass commented:
@emilianbold I added the newlines.

emilianbold commented on spring.beans/src/org/netbeans/modules/spring/beans/resources/templates/springXMLConfig.template:
```diff
@@ -1,4 +1,24 @@
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
```
Please undo the license header for this .template file and exclude it from nbbuild/build.xml for the "rat" task.

See how it was done for this other PR: https://github.com/emilianbold/incubator-netbeans/pull/282/files#diff-fa67ba03971f6d21c0ff2493bf5d922f

batouxclass replied:
Done

emilianbold commented on spring.beans/src/org/netbeans/modules/spring/beans/ui/customizer/Bundle.properties:
```diff
@@ -1,45 +1,19 @@
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
```
This is OK.

batouxclass committed:
Merge branch 'master' of https://github.com/emilianbold/incubator-netbeans into #194-Cleanup-NetBeans-license-header-for-module-spring.beans

batouxclass committed:
[NETBEANS-54] Module Review spring.beans

- Excluded the template file from nbuild/build.xml file.
- Removed the license header changes.

emilianbold committed:
Merge branch 'master' into #194-Cleanup-NetBeans-license-header-for-module-spring.beans
